### PR TITLE
example: add abacus examples

### PR DESCRIPTION
**Pull request by hongriTianqi**
Add abacus examples of fcc-Al.

### CODE REVIEW

File: examples/init/abacus/fcc-Al-lcao/machine.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
+{
+  "api_version": "1.0",
+  "deepmd_version": "2.0.1",
+  "train" :[
+    {
+      "command": "dp",
+      "machine": {
+        "batch_type": "",
```
Comment (njzjz) [inline examples/init/abacus/fcc-Al-lcao/machine.json:8]: Why is it empty?

File: examples/run/dp1.x-lammps-ABACUS-lcao-dpks/fcc-al/machine.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
+{
+  "api_version": "1.0",
+  "deepmd_version": "2.0.1",
```
Comment (njzjz) [inline examples/run/dp1.x-lammps-ABACUS-lcao-dpks/fcc-al/machine.json:3]: It's unclear why this file is put under a directory named `dp1.x`...

File: examples/init/abacus/fcc-Al-lcao/machine.json (hunk 1 above)
Comment (hongriTianqi) [reply examples/init/abacus/fcc-Al-lcao/machine.json:8]: I thought it depends on users' setting, so left it empty. Is it better to add my setting, "Lebesgue"?

File: examples/run/dp1.x-lammps-ABACUS-lcao-dpks/fcc-al/machine.json (hunk 2 above)
Comment (hongriTianqi) [reply examples/run/dp1.x-lammps-ABACUS-lcao-dpks/fcc-al/machine.json:3]: I see. I will move it to dp2.x...

**Review by njzjz — APPROVED** [review]

File: examples/run/dp2.x-lammps-ABACUS-lcao/fcc-al/run_param.json [hunk 3]
```diff
@@ -0,0 +1,110 @@
+{
```
Comment (njzjz) [inline examples/run/dp2.x-lammps-ABACUS-lcao/fcc-al/run_param.json:1]: If possible, these two files can be added to `check_examples` as well:

https://github.com/deepmodeling/dpgen/blob/369b9355d451d0e84d3deb7432970bf4d59bedb7/tests/test_check_examples.py#L33-L35

Then we do not need to check it manually.

Comment (njzjz) [reply examples/run/dp2.x-lammps-ABACUS-lcao/fcc-al/run_param.json:1]: @pxlxingliang, it looks that the variant name in #845 is incorrect, making the tests fail.

Comment (pxlxingliang) [reply examples/run/dp2.x-lammps-ABACUS-lcao/fcc-al/run_param.json:1]: > @pxlxingliang, it looks that the variant name in #845 is incorrect, making the tests fail.

We plane to modify the fp_style to be "abacus", then @hongriTianqi will push the modification.

**Review by pxlxingliang — APPROVED** [review]

**Pull request merged**